scripts/fetch_release_notes_version_name.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 Adyen N.V.
+#
+# This file is open source and available under the MIT license. See the LICENSE file for more info.
+#
+# Created by ozgur on 10/12/2024.
+#
+function fetch_release_notes_version_name() {
```

Comment:
Does this script fetch the version name from the first file found that has a name matching the regex?

I am thinking of two cases:
- What if there are more than one release notes files (since all release notes will be available on the `release-notes` branch)
- What if we want to do a hotfix, and the version name is not the highest.

Reply:
I update the script to get the name from the commit name, however that turned out to be trickier than expected since a commit message has space characters and bash splits all the words into separate arguments when it's passed to a script. As a workaround I trim all the whitespace and then pass it to the script, seems like it's working. What do you think? 🤔

Reply:
I think it would be best to not alter the commit message before processing, so we could work on the actual commit message in the script. Instead of passing it in the arguments, I would directly read the latest commit message inside the script, or add the commit message in the `env` list and use it in the script.

Reply:
Problem is when we pass the commit message as an argument to a bash command, it automatically splits the string into words and the script takes them as separate arguments which makes it difficult to parse the commit message as a whole and get the version name. That's why I think for now it's best to leave it as is, since the other alternative to it is looking at the changed files, which is also not 100% reliable.